level_02.md

```diff
@@ -91,6 +91,46 @@ Create `.eslintrc.json`:
 
 Add a lint script to run `eslint`, and then `npm run lint` (because `lint` isn't one of the standard commands, you need the extra `run`).
 
+## BDD
+
+A more familiar "user first" way of describing the behaviour of our app is
+possible with the [cypress](https://www.cypress.io) library. There's a nice
+intro on this [wanago.io blog
+post](https://wanago.io/2019/12/30/javascript-testing-introduction-end-to-end-testing-cypress/).
+
+Cypress is a sophisticated test runner, which aims to replace Selenium. It builds on
+several testing libraries including Mocha and Chai and hooks into Chrome as its
+default driver, but can also run headless -- and lots more.
+
+1. install the package with `npm install cypress --save-dev`
+2. add a `"test:features": "cypress open` script to `package.json`
```

Comment:
```suggestion
2. add a `"test:features": "cypress open"` script to `package.json`
```

Reply:
@erbridge thanks F, nice catch. Eagle eyes!

I've also spotted the use of the non-existent `context()` function on line 115 (does exist in RSpec!) and replaced that with `describe()`...